packages/airnode-protocol/contracts/rrp/authorizers/admin/interfaces/IAirnodeFeeRegistry.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+interface IAirnodeFeeRegistry {
+    event SetAirnodeEndpointFlag(
+        address airnode,
+        bool status,
+        address airnodeFlagAndPriceSetter
+    );
+
+    event SetDefaultPrice(uint256 price, address globalDefaultPriceSetter);
+
+    event SetDefaultChainPrice(
+        uint256 chainId,
+        uint256 price,
+        address globalDefaultPriceSetter
+    );
+
+    event SetDefaultAirnodePrice(
+        address airnode,
+        uint256 price,
+        address airnodeFlagAndPriceSetter
+    );
+
+    event SetDefaultChainAirnodePrice(
+        uint256 chainId,
+        address airnode,
+        uint256 price,
+        address airnodeFlagAndPriceSetter
+    );
+
+    event SetAirnodeEndpointPrice(
+        address airnode,
+        bytes32 endpointId,
+        uint256 price,
+        address airnodeFlagAndPriceSetter
+    );
+
+    event SetChainAirnodeEndpointPrice(
+        uint256 chainId,
+        address airnode,
+        bytes32 endpointId,
+        uint256 price,
+        address airnodeFlagAndPriceSetter
+    );
```

Comment:
At least `chaindId`, `airnode` and `endpointId` should be indexed in order to have them available in the "topics" for filtering.
https://docs.soliditylang.org/en/v0.8.9/contracts.html#events

Reply:
Agreed

Reply:
Also have indexed key events in the  AirnodeTokenLock which I think will be needed when calculating provider payments off chain.